doc/commands.md

```diff
@@ -128,48 +128,50 @@ Rewriting rules for definable symbols are declared using the `rule` command.
 ```
 rule add zero      $n ↪ $n
 rule add (succ $n) $m ↪ succ (add $n $m)
+rule mul zero      _  ↪ zero
 ```
 
-Note that rewriting rules can also be defined simultaneously, using the `with`
-keyword instead of the `rule` keyword for all but the first rule.
+Terms prefixed by the sigil `$` and `_` are pattern variables.
+Matching a ground term against a pattern variable always succeed.
```

Comment:
Why adding this sentence?

Reply:
To give the semantics of the pattern variables

Reply:
I think it may be confusing. I prefer to remove it. Why talking of closed terms? Even in higher order matching it's relative to a local context. Indeed, consider the following example:
```

symbol A:TYPE
symbol f:(A→A)→A
rule f(λ_,$a[]) ↪ $a
assert λx,f(λ_,x) ≡ λx,x
```

Reply:
Yes indeed, you're right: either it's not correct or it becomes too complicated. But perhaps it should be documented somewhere, how should people know about those things? Do we have any plan to provide a comprehensive documentation?

Reply:
Perhaps you can add my example in your test file?